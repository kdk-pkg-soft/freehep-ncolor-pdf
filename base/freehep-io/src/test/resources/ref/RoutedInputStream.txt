StartDThis is part of the main stream,StartAbut this is part of the A stream.EndAwhile this
is again part of the main stream.

Of course it could be more difficult:

SStStaStarStartStartBis where stream B starts
and this can be several lines long
and ends when we see EndB

and...

StartC
Lets see if this ends up again in the main stream
EndCStartE, because it should not...
EndE

and the next one should be

SClosed
closed early
EClosed

and this is the end.
EndD