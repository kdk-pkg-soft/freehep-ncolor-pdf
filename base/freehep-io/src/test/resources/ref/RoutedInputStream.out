StartDThis is part of the main stream,[StartA:StartAbut this is part of the A stream.EndA]while this
is again part of the main stream.

Of course it could be more difficult:

SStStaStarStart[StartB:StartBis where stream B starts
and this can be several lines long
and ends when we see EndB]

and...

[StartC:StartC
Lets see if this ends up again in the main stream
EndC]StartE, because it should not...
EndE

and the next one should be

[EarlyClosed:SClose]

and this is the end.
EndD